### PR TITLE
Consolidate depproj for S.Runtime.InteropServices

### PR DESCRIPTION
**Pull request by ericstj**
Fixes https://github.com/dotnet/corefx/issues/9604

/cc @weshaggard @tijoytom

### CODE REVIEW

File: src/System.Runtime.InteropServices/src/redist/System.Runtime.InteropServices.depproj [hunk 1]
```diff
@@ -5,8 +5,12 @@
   </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.20.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(TargetGroup)' == 'netcore50' OR '$(TargetGroup)' == 'netcore50aot'">4.0.20.0</AssemblyVersion>
```
Comment (weshaggard) [inline src/System.Runtime.InteropServices/src/redist/System.Runtime.InteropServices.depproj]: Does the depproj even need the AssemblyVersion metadata? Is it only for the packaging validation?

File: src/System.Runtime.InteropServices/src/redist/project.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-    "System.Runtime.InteropServices": "4.0.20"
+    "System.Runtime.InteropServices": "4.1.0"
   },
   "frameworks": {
-    "netcore50": {}
+    "netcore50": {},
+    "net462": {}
   },
   "runtimes": {
+    "win": {},
```
Comment (weshaggard) [inline src/System.Runtime.InteropServices/src/redist/project.json]: why do we need win?

**Comment by weshaggard** [comment]
LGTM

File: src/System.Runtime.InteropServices/src/redist/System.Runtime.InteropServices.depproj (hunk 1 above)
Comment (ericstj) [inline src/System.Runtime.InteropServices/src/redist/System.Runtime.InteropServices.depproj]: Yes.

**Pull request merged**